README.md

```diff
@@ -110,6 +110,14 @@ pip install scalene
 scalene opentelemetry-<PACKAGE>/tests/performance/resource-usage/<PATH_TO_TEST>/profile_resource_usage_<NAME_OF_TEST>.py
 ```
 
+## Python Version Support Addition and Removal
+
+This project supports the latest Python versions. As new Python versions are released, support for them is added and
+as Python versions are deprecated, support for them is removed.
```

Comment:
deprecated or "EOLed"?

Reply:
Might be good to link to here https://devguide.python.org/devcycle/#end-of-life-branches

Reply:
I have removed `are deprecated` for `reach their end of life`, that should be more accurate, thanks @owais :+1: 

I have also added the link above, @codeboten :+1: